src/main/resources/db/migration/V1_6_Create_daily_list_table.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS moj_daily_list (
```

Comment:
I think this file needs to have a double underscore in the name like the others?
So it would be `V1_6__create_daily_list_table.sql`